packages/babel-preset-env/data/built-in-modules.json

```diff
@@ -6,7 +6,7 @@
     "safari": "10.1",
     "opera": "48",
     "ios_saf": "10.3",
-    "and_chr": "71",
-    "and_ff": "64"
+    "and_chr": "74",
```

Comment:
These shouldn't be bumped since caniuse only keeps lastest mobile browser version.

Reply:
Oops! I just followed the instructions outlined in the linked [issue](https://github.com/babel/babel/issues/10809) and this got bumped when I ran `yarn build-data`. Even without any of the other changes, running `yarn build-data` seems to auto bump these. Is that expected? I'll revert this file manually for now.

Reply:
Yeah please revert it manually; we'll have to check why the build script updated it.

Reply:
The build script is fixed in https://github.com/babel/babel/pull/10814